.github/actions/setup-miniconda/action.yml

```diff
@@ -40,7 +41,7 @@ runs:
         uses: actions/cache@v3
         with:
           path: ${{ runner.temp }}/miniconda
-          key: miniconda-${{ runner.os }}-${{ runner.arch }}-${{ inputs.python-version }}-${{ steps.get-date.outputs.today }}
+          key: miniconda-${{ inputs.miniconda-version }}-${{ runner.os }}-${{ runner.arch }}-${{ inputs.python-version }}-${{ steps.get-date.outputs.today }}
```

Comment:
This is to invalidate the current cache.  In hindsight, the conda version should be part of the key anyway.